src/legacy/core_plugins/kibana/server/tutorials/apache_logs/index.js

```diff
@@ -36,25 +36,25 @@ export function apacheLogsSpecProvider(server, context) {
       defaultMessage: 'Collect and parse access and error logs created by the Apache HTTP server.',
     }),
     longDescription: i18n.translate('kbn.server.tutorials.apacheLogs.longDescription', {
-      defaultMessage: 'The apache2 Filebeat module parses access and error logs created by the Apache 2 HTTP server. \
+      defaultMessage: 'The apache Filebeat module parses access and error logs created by the Apache HTTP server. \
 [Learn more]({learnMoreLink}).',
       values: {
-        learnMoreLink: '{config.docs.beats.filebeat}/filebeat-module-apache2.html',
+        learnMoreLink: '{config.docs.beats.filebeat}/filebeat-module-apache.html',
       },
     }),
     euiIconType: 'logoApache',
     artifacts: {
       dashboards: [
         {
-          id: 'Filebeat-Apache2-Dashboard',
+          id: 'Filebeat-Apache-Dashboard-ecs',
```

Comment:
Can you double check if we renamed the ID in the Beats repo for the dashboard?

Reply:
@ruflin Yeah I tested this and it works, also found the id name in filebeat https://github.com/elastic/beats/blob/master/filebeat/module/apache/_meta/kibana/7/dashboard/Filebeat-apache.json#L647